h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -669,6 +709,7 @@ private void validateModelCategory(ModelCategory c) throws PredictException {
           }
           if (levelIndex == null) {
             if (convertUnknownCategoricalLevelsToNa) {
+              errorConsumer.dataTransformError(dataColumnName, o, "Unknown categorical level detected, converting to NaN.");
```

Comment:
Should this be a different method? The user explicitly set "convertUnknownCategoricalLevelsToNa" - it is expected behavior. Might make sense to treat it as a different event.

Reply:
This error will be moved somewhat lower, to the "else" statement, where I find it missing. A new level just for categoricals was introduced.